src/url-replacements.js

```diff
@@ -25,3 +29,13 @@ export function urlReplacementsForDoc(nodeOrDoc) {
   return /** @type {!./service/url-replacements-impl.UrlReplacements} */ (
       getExistingServiceForDocInEmbedScope(nodeOrDoc, 'url-replace'));
 }
+
+/**
+ * @param {!./service/ampdoc-impl.AmpDoc} ampdoc
+ * @param {!Window} embedWin
+ * @param {*} varSource
+ */
+export function installUrlReplacementsForEmbed(ampdoc, embedWin, varSource) {
```

Comment:
This method should go to the `url-replacements-impl`. Why? To avoid any risk of all other extensions compiling the whole `UrlReplacements` in. Only whoever needs it will compile it by depending on `url-replacements-impl` directly. Does it make sense?